nemo/collections/nlp/data/datasets/datasets_utils/data_preprocessing.py

```diff
@@ -77,7 +79,11 @@ def get_label_stats(labels, outfile='stats.tsv'):
         if i < 3:
             logging.info(f'{i} item: {k}, {v} out of {total}, {v / total}.')
         i += 1
-    return total, label_frequencies
+
+    freq_dict = {}
+    for lf in label_frequencies:
+        freq_dict[lf[0]] = lf[1]
```

Comment:
you can add this to the previous for loop

Reply:
Merged them.